system/hrpc/hrpc.cpp

```diff
@@ -328,11 +326,14 @@ void HRPCcommon::doproxy(HRPCcallframe &frame,int fn,IHRPCtransport *tr,HRPCbuff
         buff.releasewrite(base);
         int timeleft=-1;
         HRPCtrace(" doproxy pre-receive\n");
-        if (timelimit) {
-            timeleft = (duetime-msTick());
-            if (timeleft<=0) {
+        if (timelimit) 
+        {
+            unsigned now = msTick();
+            if (now - start > timelimit)
```

Comment:
I think this needs to be >= rather than >

Reply:
Fixed and repushed